pkg/events/events.go

```diff
@@ -5767,6 +5768,27 @@ var Definitions = eventDefinitions{
 				{Type: "const char*const*", Name: "symbols"},
 			},
 		},
+		SymbolsCollision: {
+			ID32Bit: sys32undefined,
+			Name:    "symbols_collision",
+			Probes:  []probeDependency{},
+			Dependencies: dependencies{
+				Events: []eventDependency{
+					{EventID: SharedObjectLoaded},
+					{EventID: SchedProcessExec}, // Used to get mount namespace cache
+				},
+				Capabilities: []cap.Value{
+					cap.SYS_PTRACE,   // Used to get host mount NS for bucket cache
```

Comment:
I'm afraid you will have to check the run with: `--capabilities bypass=false` and, likely ?, raise privileges (ring1) in the places where the privilege is needed (that's a good question for the other bucket cache PR as well.

Reply:
I have created https://github.com/aquasecurity/tracee/pull/2380 to fix a few procfs bad permissions we were having.